venues/ICLR.cc/2019/Conference/webfield/homepage.js

```diff
@@ -22,112 +22,25 @@ var REVIEWERS_ID = CONFERENCE_ID + '/Reviewers';
-  'Applications: Sustainability and Climate',
-  'Applications: Text and Web Data',
-  'Applications: User Models',
-  'Applications: Vision',
-  'Data: Big Data',
-  'Data: Multivariate',
-  'Data: Other',
-  'Data: Relational',
-  'Data: Spatial',
-  'Data: Temporal or Sequential',
-  'Learning: Active Learning',
-  'Learning: Classification',
-  'Learning: Clustering',
-  'Learning: Deep Learning',
-  'Learning: General',
-  'Learning: Nonparametric Bayes',
-  'Learning: Online and Anytime Learning',
-  'Learning: Other',
-  'Learning: Parameter Estimation',
-  'Learning: Probabilistic Generative Models',
-  'Learning: Ranking',
-  'Learning: Recommender Systems',
-  'Learning: Regression',
-  'Learning: Reinforcement Learning',
-  'Learning: Relational Learning',
-  'Learning: Relational Models',
-  'Learning: Scalability',
-  'Learning: Semi-Supervised Learning',
-  'Learning: Structure Learning',
-  'Learning: Structured Prediction',
-  'Learning: Theory',
-  'Learning: Unsupervised',
-  'Methodology: Bayesian Methods',
-  'Methodology: Calibration',
-  'Methodology: Elicitation',
-  'Methodology: Evaluation',
-  'Methodology: Human Expertise and Judgement',
-  'Methodology: Other',
-  'Methodology: Probabilistic Programming',
-  'Models: Bayesian Networks',
-  'Models: Directed Graphical Models',
-  'Models: Dynamic Bayesian Networks',
-  'Models: Markov Decision Processes',
-  'Models: Mixed Graphical Models',
-  'Models: Other',
-  'Models: Relational Models',
-  'Models: Topic Models',
-  'Models: Undirected Graphical Models',
-  'None of the above',
-  'Principles: Causality',
-  'Principles: Cognitive Models',
-  'Principles: Decision Theory',
-  'Principles: Game Theory',
-  'Principles: Information Theory',
-  'Principles: Other',
-  'Principles: Probability Theory',
-  'Principles: Statistical Theory',
-  'Representation: Constraints',
-  'Representation: Dempster-Shafer',
-  'Representation: Fuzzy Logic',
-  'Representation: Influence Diagrams',
-  'Representation: Non-Probabilistic Frameworks',
-  'Representation: Probabilistic'
-];
-
 var HEADER = {
   title: 'ICLR 2019',
   subtitle: 'International Conference on Learning Representations',
   location: 'New Orleans, Louisiana, United States',
   date: 'May 6 - May 9, 2019',
   website: 'https://iclr.cc/Conferences/2019',
-  instructions: '<p><strong>Important note to Authors and Reviewers: Please update your OpenReview profile to have all your recent emails</strong></p>\
-    <p><strong>Important Information</strong><br>\
-    Note that ICLR 2019 Conference submissions are now open.</p> \
-    <p><strong>Questions or Concerns</strong><br> \
-    Please contact the OpenReview support team at \
-    <a href="mailto:info@openreview.net">info@openreview.net</a> with any questions or concerns about the OpenReview platform. \</br> \
+  instructions: '<p><strong>Important note to Authors and Reviewers: Please update your OpenReview profile to have all your recent emails.</strong></p>\
```

Comment:
suggestion: you could put the "Please update your OpenReview profile..." message in the "Important Information" list. Right now it feels a little redundant, since there is "Important note to Authors and Reviewers" *and* "Important information". If you do this, the "Please update your OpenReview profile..." message should be the first item in the list, in my opinion.